storm-core/src/ui/public/templates/topology-page-template.html

```diff
@@ -313,6 +320,10 @@
         </th>
         <th class="header">Last error
         </th>
+		<th class="header">
```

Comment:
@caofangkun 
Before that we need to establish code style for Java, HTML, etc. And have formatter file to let users applies it easily.

For example, Jedis uses Eclipse Java formatter which HBase project is using.
https://github.com/xetorthio/jedis/blob/master/hbase-formatter.xml
And ```make format``` formats whole files, so any contributors can run that commands before commit.